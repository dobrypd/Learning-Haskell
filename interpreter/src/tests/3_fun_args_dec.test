int a( int a, int b) {
    return;
}
int b(bool b) {
    return 1;
}
bool c(int a, bool z) {
    return;
}
