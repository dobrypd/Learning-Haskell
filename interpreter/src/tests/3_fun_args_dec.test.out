Welcome to Kappa interpreter
undefined
[(fromList [],fromList [(Ident "a",([Ident "a",Ident "b"],[SStm (JumpS SjumpFour)])),(Ident "b",([Ident "b"],[SStm (JumpS (SjumpFive (Econst (Eint 1))))])),(Ident "c",([Ident "a",Ident "z"],[SStm (JumpS SjumpFour)]))],fromList [])]
